lib/virt_autotest/common.pm

```diff
@@ -153,6 +153,24 @@ if (get_var("REGRESSION", '') =~ /xen/) {
             distro       => 'SLE_15',
             location     => 'http://mirror.suse.cz/install/SLP/SLE-15-SP2-Full-GM/x86_64/DVD1/',
         },
+        sles15sp3PV => {
+            name         => 'sles15sp3PV',
+            autoyast     => 'autoyast_xen/sles15sp3PV_PRG.xml',
+            extra_params => '--os-variant sles12',
```

Comment:
We probably should use `sle15sp3` and also update the other ones. I not possible at least reference a bsc number here why we need to use a old version.

Reply:
OK, this is actually bigger issue:

The problem is that on older hosts we don't have those newer options.
I suggest to create a ticket and discuss / plan / track the issue separately.

Reply:
For very old hosts that are LTSS for a long time already, maybe such workaround can be used. But for newer products, eg sle12sp4/5, sle15spx, the correct os-variant should be there, right? If not, some bug should be opened?

Reply:
I've created https://progress.opensuse.org/issues/92782 for it, so I guess we can leave this as it is and find a solution elsewhere.

Reply:
I'm marking this as resolved here as we have separate ticket for this.